lib/us_web_design_standards/sass.rb

```diff
@@ -2,7 +2,7 @@
 
 module USWebDesignStandards
   BASEDIR = File.dirname File.dirname(File.dirname(__FILE__))
-  SASS_DIR = File.join BASEDIR, 'assets', '_scss'
+  SASS_DIR = File.join BASEDIR, 'assets', 'sass'
```

Comment:
Standardizing the directory structure to be `sass` instead of `_scss` which was a legacy directory name because it was imported from Jekyll.